posh-svn.psd1

```diff
@@ -0,0 +1,56 @@
+@{
+
+# Script module or binary module file associated with this manifest.
+ModuleToProcess = 'posh-svn.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.7.301'
+
+# ID used to uniquely identify this module
+GUID = 'f18820b6-2e02-41b4-afc5-de886bb1b848'
+
+# Author of this module
+Author = 'Chris Donnelly, Jeremy Skinner and contributors'
+
+# Copyright statement for this module
+Copyright = '(c) 2010-2017 Chris Donnelly, Jeremy Skinner and contributors'
```

Comment:
We don't have a copyright in place and would actually prefer that it be eliminated for this project.

Reply:
Sure.  Just force of habit, will remove.